zipline-bytecode/src/test/kotlin/app/cash/zipline/bytecode/LineNumbersTest.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.bytecode
+
+import com.google.common.truth.Truth.assertThat
+import okio.Buffer
+import okio.ByteString.Companion.decodeHex
+import org.junit.Test
+
+class LineNumbersTest {
```

Comment:
It's kind of a change detector test, but I would feel more comfortable with a test which used a real `QuickJs` to produce a table, round-trip it through reader/writer, and validate it produces the same `ByteString`.

Reply:
YES! We have that in JsObjectEncodingTest, which has a function, `assertRoundTrip()`.

Reply:
Oh, but not for line numbers. Yeah, good call. Lemme add that too.